yb-voyager/src/query/queryissue/issues_ddl_test.go

```diff
@@ -341,6 +360,39 @@ func testUniqueNullsNotDistinctIssue(t *testing.T) {
 	assertErrorCorrectlyThrownForIssueForYBVersion(t, err, "syntax error", uniqueNullsNotDistinctIssue)
 }
 
+func testBeforeRowTriggerOnPartitionedTable(t *testing.T) {
```

Comment:
was this just a miss?

Reply:
the issue was PG12 feature and added before we had this issues-intergrations pipeline so we didn't had the test case for it but this is fixed in 2.25 as I can see on the docs, so added test case and marked it as fixed.

Reply:
Awesome thanks!